frontend/packages/design-system/src/themes/Button.ts

```diff
@@ -3,7 +3,7 @@ import textTheme from './Text';
 export default {
   baseStyle: {
     _text: {
-      ...textTheme.variants.button,
+      ...textTheme.variants.heading4,
```

Comment:
I'm not sure the button text should be linked to the heading 4, unless there is a reason to.